Project 2/index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--sources
+image:https://www.owame.com/products/japanese-mori-girl-long-skirts-women-high-waist-army-green-umbrella-skirts-pocket-female-cotton-midi-saia-solid-color
+fonts: cambria and gill sans
+-->
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <section class="clearfix">
+
+        <img src="images/Capture.JPG" alt="green A line skirt" class="left">
+        <div class="right">
+            <h2>Stylish Flowing Skirt</h2>
+            <div class="price">Price: $12.99</div>
+
+            <svg id="favorite" data-name="favorite" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 97 97">
```

Comment:
Both these SVGs are actions on the page, so they should be wrapped in either `a` or `button` elements, depending on if they open links to additional web pages or perform actions on this page. This would also prompt you to add hover/focus effects to these elements 👍